app/controllers/api/v1/project_roles_controller.rb

```diff
@@ -10,9 +10,13 @@ def resource_name
   end
 
   def update
-    super do
-      UserAddedToProjectMailerWorker.perform_async(api_user.id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
-    end
+    UserAddedToProjectMailerWorker.perform_async(acl_user_id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
```

Comment:
Ah hah. It looks like there's an exception being raised in CreatableResource that isn't bubbling: `#<ActiveRecord::RecordInvalid: Validation failed: User group . Roles have already been set for this user or group>`. Because I was manually creating the ACLs in the let block. Which of course I wanted--nay, _needed_ to have for #update, but were severely getting in the way on a #create.